src/client/Table.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+
+function TableItem(props){
+  const isEven = (props.seq % 2 === 0) ? true : false;
+  const className = isEven ? 'table__row table__row--even' : 'table__row table__row--odd';
+
+  return(
+    <tr className={className}>
+        <td>{props.seq}</td>
+        <td>{props.status}</td>
+        <td>{props.category}</td>
+        <td>{props.title}</td>
+        <td>{props.owner}</td>
+        <td>{props.priority}</td>
+      </tr>
+  )
+}
+
+function ShowTable(props){
```

Comment:
To be honest, `<ShowTable />` component is redundant. As [thinking-in-react](https://reactjs.org/docs/thinking-in-react.html#step-1-break-the-ui-into-a-component-hierarchy) mentioned, the first thing we should consider is how to draw boxes around our table. Take `table` for example, we may consider to modulize the table to `table`(you've already done), `row` or `cell`.

However, one `table` one component or dividing table `row` into a component are feasible. Separating UI into components depends on what kind of graininess do we want.